common/include/common/common_utils.hpp

```diff
@@ -78,8 +83,9 @@ using tbmv_param_t =
 using trsv_param_t = std::tuple<std::string, std::string, std::string, index_t>;
 
 template <typename scalar_t>
-using trsm_batched_param_t = std::tuple<char, char, char, char, index_t,
-                                        index_t, scalar_t, index_t, int>;
+using trsm_batched_param_t =
```

Comment:
This change will break the cuBLAS `trsm_batched` [benchmark](https://github.com/codeplaysoftware/sycl-blas/blob/master/benchmark/cublas/blas3/trsm_batched.cpp#L223) if merged.